src/sentry/release_health/metrics.py

```diff
@@ -170,6 +171,17 @@ def _get_crash_free_rate_data(
             project_id = get_path(group, "by", "project_id")
             assert project_id is not None
             totals = get_path(group, "totals", "rate")
+            if totals is None:
+                logger.info(
```

Comment:
Just to be overcautious, should we wrap this in try/except? Will help us avoid any problems with json encoding if there are any